### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by waithethtetoo**
add dependencies ,

### CODE REVIEW

File: PADC-FP-GoldenMyanmarTour/app/src/main/java/com/padc/goldenmyanmartour/activity/HomeActivity.java [hunk 1]
```diff
@@ -1,33 +1,103 @@
...
+    DrawerLayout drawerLayout;
+
+    @BindView(R.id.toolbar)
+    Toolbar toolbar;
+
+    @BindView(R.id.navigation_view)
+    NavigationView navigationView;
+
+    @BindView(R.id.pager_images)
+    ViewPager pager_images;
```
Comment (vincent-paing) [inline PADC-FP-GoldenMyanmarTour/app/src/main/java/com/padc/goldenmyanmartour/activity/HomeActivity.java]: Doesn't follow Naming Convention.

Comment (vincent-paing) [inline PADC-FP-GoldenMyanmarTour/app/src/main/java/com/padc/goldenmyanmartour/activity/HomeActivity.java]: Doesn't follow naming convention

File: PADC-FP-GoldenMyanmarTour/app/src/main/java/com/padc/goldenmyanmartour/activity/HomeActivity.java [hunk 2]
```diff
@@ -1,33 +1,103 @@
...
+
+    @BindView(R.id.pi_image_slider)
+    PageIndicatorView pi_image_slider;
```
Comment (vincent-paing) [inline PADC-FP-GoldenMyanmarTour/app/src/main/java/com/padc/goldenmyanmartour/activity/HomeActivity.java]: Doesn't follow naming convention

File: PADC-FP-GoldenMyanmarTour/app/src/main/java/com/padc/goldenmyanmartour/activity/HomeActivity.java (hunk 1 above)
Comment (agpyaephyo) [reply PADC-FP-GoldenMyanmarTour/app/src/main/java/com/padc/goldenmyanmartour/activity/HomeActivity.java]: pager_images should be pagerImages. the rest is fine.

File: PADC-FP-GoldenMyanmarTour/app/src/main/java/com/padc/goldenmyanmartour/activity/HomeActivity.java (hunk 2 above)
Comment (agpyaephyo) [reply PADC-FP-GoldenMyanmarTour/app/src/main/java/com/padc/goldenmyanmartour/activity/HomeActivity.java]: the same for pi_image_slider.

**Comment by agpyaephyo** [comment]
@waithethtetoo This PR is from **develop** to **master**. This is wrong. The PR should be from your feature branch to **develop** branch. Didn't you create a feature branch for your implementation ?

File: PADC-FP-GoldenMyanmarTour/app/src/main/java/com/padc/goldenmyanmartour/activity/HomeActivity.java (hunk 1 above)
Comment (waithethtetoo) [reply PADC-FP-GoldenMyanmarTour/app/src/main/java/com/padc/goldenmyanmartour/activity/HomeActivity.java]: Yes Teachers

**Pull request closed without merging**